hooks/environment

```diff
@@ -4,10 +4,10 @@
 # 1. Ensure that the requested keychain exists
 # 2. Validate that this job is allowed to proceed with codesigning tasks.
 
+set -o errexit -o nounset -o pipefail
 [[ -n "${DEBUG-}" ]] && set -x
 
-# TODO: Make this be the actual signing host, not my laptop.
-permitted_signing_host="sean-mac-c02xv3a8jgh6"
+permitted_signing_host="codesigning-agent"
```

Comment:
Hah, I forgot that I had never fixed this from my test value. Whoops.

Reply:
Please burn your laptop now.

Reply:
> Please burn your laptop now.

Awwwww, I just got rid of traps. :(